examples/next/components/Header.tsx

```diff
@@ -2,9 +2,11 @@ import { Button, useColorMode } from 'theme-ui'
 
 const Header = () => {
   const [colorMode, setColorMode] = useColorMode()
+
   return (
     <header>
       <Button
+        suppressHydrationWarning
```

Comment:
When I launched the Next example, I noticed the hydration warning popping up straight in your face. This component _will_ cause it, because it's using `useColorMode` without waiting until we're on the client, and we depend on the color scheme value in local storage. (We could render a skeleton for the button on the server instead, but that's not much better UX IMHO.)